hedera-node/cli-clients/src/main/java/com/hedera/services/cli/sign/SignUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.cli.sign;
+
+import java.nio.ByteBuffer;
+
+public class SignUtils {
+    /**
+     * next bytes are signature
+     */
+    public static final byte TYPE_SIGNATURE = 3;
```

Comment:
Why are we using `TYPE_*` here ?

Reply:
this type will be used for record stream